gpu/README.md

```diff
@@ -1,10 +1,10 @@
-# Dataproc - GPU driver installation
+# GPU driver installation
 
-GPUs require special drivers and software. These items are not pre-installed on
-Cloud Dataproc clusters by default. This
-[initialization action](https://cloud.google.com/dataproc/init-actions) installs
-GPU drivers for NVIDIA on master and workers node in a
-[Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+GPUs require special drivers and software which are not pre-installed on
+[Cloud Dataproc](https://cloud.google.com/dataproc) clusters by default.
+Initialization actions in this (https://cloud.google.com/dataproc/init-actions)
```

Comment:
nit: this sentence doesn't read good, maybe just use "This initialization action installs"?

Reply:
Done.